gradle/plugins/src/main/kotlin/com.hedera.gradle.protobuf.gradle.kts

```diff
@@ -35,6 +35,8 @@ protobuf {
 
 configurations.configureEach {
     if (name.startsWith("protobufToolsLocator") || name.endsWith("ProtoPath")) {
+        @Suppress("UnstableApiUsage")
+        shouldResolveConsistentlyWith(configurations.getByName("mainRuntimeClasspath"))
```

Comment:
Just curious, why is this necessary?

Reply:
This is how @jjohannes fixed an issue caused by a corner case of a "prod" code (`PlatformTestingTool` module) depending on a test module `swirlds-platform-core-test-fixture`. It doesn't work without this code and fails with a really obscure error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':PlatformTestingTool:extractIncludeProto'.
> Could not resolve all dependencies for configuration ':PlatformTestingTool:compileProtoPath'.
   > Could not find org.junit.jupiter:junit-jupiter-params:.
     Required by:
         project :PlatformTestingTool > project :swirlds-platform-core > project :swirlds-state-impl

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Get more help at https://help.gradle.org/.

BUILD FAILED in 6s
```

Reply:
Ok, thanks for the context!